code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2843,7 +2843,15 @@
 		if(15 to 35)
 			M.adjustCloneLoss(-2)
 			M.heal_organ_damage(2, 1)
-			M.status_flags &= ~DISFIGURED
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				var/datum/organ/external/head/head_organ = H.get_organ(LIMB_HEAD)
+				if(head_organ.disfigured)
+					head_organ.disfigured = FALSE
+					if(H.get_face_name() != "Unknown")
+						H.visible_message("<span class='notice'>[H]'s face shifts and knits itself back into shape!</span>","<span class='notice'>You feel your face shifting and repairing itself!</span>")
+					else if(!(H.stat == UNCONSCIOUS || H.sleeping > 0))
```

Comment:
surely if sleeping is > 0 then you will also be unconscious, yes?

Reply:
It's the same as what's in the code for seeing emotes and I didn't want to check for sight there. For all I know there's some weird edge case. Maybe I should figure this out though.
https://github.com/vgstation-coders/vgstation13/blob/242178dcef5fd87c7e575dea39f7d5ae8889ac71/code/modules/mob/mob.dm#L352

Reply:
Just use `isUnconscious` and call it a day